### PR TITLE
DOCS-5839 clarify SNMP trap and bind concerns

### PR DESCRIPTION
**Pull request by andf-mongodb**

### CODE REVIEW

File: source/includes/steps-monitor-with-snmp.yaml [hunk 1]
```diff
@@ -1,32 +1,177 @@
...
 stepnum: 1
 ref: copy-config-files
-pre: |
-  Use the following sequence of commands to move the SNMP
-  configuration files to the SNMP service configuration directory.
+content: |
+  Copy the MongoDB SNMP configuration files from their installation
+  location to their target directories, according to your platform:
+  
+  .. tabs::
```
Comment (kay-kim) [inline source/includes/steps-monitor-with-snmp.yaml]: Heh -- this is actually included in Monitor MongoDB **Windows** with SNMP page

https://docs-mongodbcom-staging.corp.mongodb.com/andrew.feierabend/DOCS-5839-clarify-snmp-trap-and-bind/tutorial/monitor-with-snmp-on-windows.html

File: source/includes/fact-snmp-configuration-files.rst [hunk 2]
```diff
@@ -1,18 +1,50 @@
...
 
   The configuration file to run |mongod-program| as the SNMP
-  master. This file sets SNMP run-time configuration options.
+  master. This file sets SNMP run-time configuration options,
+  including the ``agentaddress`` to run on.
+
+The MongoDB SNMP configuration files are provided with your MongoDB
+Enterprise installation, as follows:
+
+- If you installed MongoDB Enterprise via a package manager, these files
```
Comment (kay-kim) [inline source/includes/fact-snmp-configuration-files.rst]: Also included in the SNMP MongoDB Windows page

https://docs-mongodbcom-staging.corp.mongodb.com/andrew.feierabend/DOCS-5839-clarify-snmp-trap-and-bind/tutorial/monitor-with-snmp-on-windows.html#configuration-files

File: source/includes/steps-monitor-with-snmp.yaml (hunk 1 above)
Comment (andf-mongodb) [reply source/includes/steps-monitor-with-snmp.yaml]: You're right. Fixed by moving the Windows-specific copy to its own file.

File: source/includes/fact-snmp-configuration-files.rst (hunk 2 above)
Comment (andf-mongodb) [reply source/includes/fact-snmp-configuration-files.rst]: You're right. Fixed by moving the Windows-specific instructions to their own file. Will revisit in a separate ticket.

**Pull request merged**